plugins/woocommerce-blocks/assets/js/blocks/product-filters/index.tsx

```diff
@@ -31,35 +31,18 @@ if ( isExperimentalBuild() ) {
 				/>
 			),
 		},
+		edit,
+		save,
 		variations: [
```

Comment:
![image](https://github.com/woocommerce/woocommerce/assets/5423135/26b0d8e9-32ac-4115-84a9-a2ebf91d4042)

We can use `isActive` callback here to use the variation title: https://developer.wordpress.org/block-editor/reference-guides/block-api/block-variations/#using-isactive

Reply:
@dinhtungdu Can you explain why we should do that? I don't really see it. This could be called "Product Filter" or "Product Filter Group", or something like that, but since this is a wrapper block why does naming it after the filter make it clearer or better from a UI/UX perspective?

Reply:
Thanks for raising this! As discussed in dm, let's go with `Product Filter`. The intention was to avoid introducing `wrapper` to users, but adding `controls` to filter block isn't better.